testsuite/Rakefile

```diff
@@ -14,12 +14,11 @@ namespace :cucumber do
       timestamp = Time.now.strftime('%Y%m%d%H%M%S')
       json_results = "-f json -o output_#{timestamp}-#{filename}.json"
       html_results = "-f html -o output_#{timestamp}-#{filename}.html"
-      profiles = ENV['PROFILE'] || 'default'
-      include_profiles = ""
-      profiles.split(',').each do |profile|
-        include_profiles << "--profile #{profile} "
-      end
-      cucumber_opts = %W[#{html_results} #{json_results} #{junit_results} -f rerun #{include_profiles} --out failed.txt -f pretty -r features]
+      profiles = ENV['PROFILE']
+      include_profiles = profiles.nil? ? '--profile default' : '--profile' + profiles.gsub(',', ' --profile ')
```

Comment:
Do we need white space after the first `--profile`?

Reply:
Oups, yes, we need it.